appveyor-3.6.yml

```diff
@@ -37,12 +37,10 @@ before_build:
   - rake "prepare_for_build[%APPVEYOR_BUILD_VERSION%]"
 
 build_script:
-  - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1-R3/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
-  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip', repos = NULL, type = 'binary')"
-  - Rscript -e "install.packages(c('ggplot2', 'patchwork', 'vdiffr'), repos = 'http://cran.us.r-project.org')"
-  - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/tlf-library/artifacts/tlf.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages(c('testthat', 'vdiffr', 'spelling','covr'), repos = 'http://cran.us.r-project.org')"
```

Comment:
this can be moved into `install_OSP_Packages.R` as well.

Reply:
Done in 26d8f56